jax/_src/numpy/array_methods.py

```diff
@@ -803,8 +810,9 @@ def set(self, values, *, indices_are_sorted=False, unique_indices=False,
                                    unique_indices=unique_indices, mode=mode,
                                    out_sharding=out_s)
 
-  def apply(self, func, *, indices_are_sorted=False, unique_indices=False,
-            mode=None):
+  def apply(self, func: Callable[[ArrayLike], Array], *,
+            indices_are_sorted: bool = False, unique_indices:bool = False,
```

Comment:
Nit: add a space after `unique_indices:` here and below.

Reply:
Done